src/services/User.ts

```diff
@@ -0,0 +1,70 @@
+import User from '../model/user';
+import { Op } from 'sequelize';
+
+export default class UserService {
+    public async getUsers<T>(login: T, limit: T): Promise<void> {
```

Comment:
Strange types. For me, string for login and number for limit looks great

Reply:
but if set string and number ws highlight error and suggests next 'string | string[] | QueryString.ParsedQs | QueryString.ParsedQs[] | undefined'